org.lyx.LyX.LilyPond.yml

```diff
@@ -0,0 +1,79 @@
+id: org.lyx.LyX.LilyPond
+runtime: org.lyx.LyX
+runtime-version: 'beta'
```

Comment:
Why beta?

Reply:
I like to verify that it works before moving to the main branch. Also in the beta branch the mount points are specified. I don't know if the latter is important.

Reply:
OK, that's fine, then it needs to be merged to `beta` branch. Also you should define your extension points a bit generically https://github.com/flathub/org.lyx.LyX/blob/540d8c97183e3ff185535469632dbe707afa6acf/org.lyx.LyX.yml#L23C5-L32

should be `org.lyx.LyX.Plugin`, then you can submit extensions like this as `org.lyx.LyX.Plugin.LilyPond`, otherwise every extension will need an extension point.

Reply:
Yes, those changes had already been proposed by Hfiguiere. I have not done it because I would have to change the extension identifier and modify the beta branch.
In any case, I will respond to the suggestions.